modules/react/modal/stories/examples/BodyOverflow.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import {Modal} from '@workday/canvas-kit-react/modal';
+import {DeleteButton} from '@workday/canvas-kit-react/button';
+import {HStack, VStack} from '@workday/canvas-kit-labs-react';
+import {space} from '@workday/canvas-kit-react/tokens';
+
+export const BodyOverflow = () => {
+  const handleDelete = () => {
+    console.log('Deleted item');
+  };
+
+  return (
+    <Modal>
+      <Modal.Target as={DeleteButton}>Delete Item</Modal.Target>
+      <Modal.Overlay>
+        <Modal.Card as={VStack} spacing={0} maxHeight={`calc(100vh - ${space.xl} * 2)`}>
```

Comment:
Question is should `Modal.Card` always be a columnar flex container? This would be a breaking change, but would allow for automatic overflow handling without a the large amount of work otherwise. For example, this example works, but isn't compliant with the different Modal card margins depending on screen size. That's probably better handled by the component.

Reply:
Most content is in a column right? I think that makes sense even though it might be a breaking change

Reply:
Especially if consumers can use or Hstack and styled properties to override things, it think it should be fine

Reply:
I mean the Modal is a `VStack` by default. A modal is almost always a vertical stack of a header, body, and optional footer. Modal variants also only add to vertical stack elements: image, header, body, optional footer. It would probably break a lot of modals. The change would be from static block positioning (block elements naturally stack vertically) to flexbox vertical stacking. The problem might be how spacing is handled.

A naive approach might be built-in spacing with the option to override. This example adds no spacing since `Heading` has margins built in (would be removed). You can see `paddingTop` added to the `HStack` later. Basically building in spacing to `VStacks` would require an overhaul to built-in margin/padding of components. The transition from children control layout spacing to containers controlling layout spacing.

Reply:
These examples show how different overflow can be achieved by direct manipulation of styling of the subcomponents. It would be nice for some styling to be built-in, but that might cause cascading breaking changes.

Reply:
I updated the `Popup.Card` to be a flex container and the complexity of this example is no longer required.